backoff.go

```diff
@@ -0,0 +1,68 @@
+package grpc
+
+import (
+	"math/rand"
+	"time"
+)
+
+// DefaultBackoffConfig uses values specified for backoff in
+// https://github.com/grpc/grpc/blob/e1d8d4dc1bdf54273b7ed9d4a495789430cbdf6c/doc/connection-backoff.md.
```

Comment:
Use this link?
https://github.com/grpc/grpc/blob/master/doc/connection-backoff.md